### PR TITLE
added an .env.example file and better event handling structure

### PR DESCRIPTION
**Pull request by reddiedev**
added an env template for relatively faster and easier project setup for collaborators

### CODE REVIEW

**Comment by reddiedev** [comment]
I just added this one here separately, as you have suggested. I'll add the events folder/files revamp in the next PR!

**Comment by reddiedev** [comment]
## one file for each event
makes it easier to modify and add more events in the future, since you only need to worry about one event at a time. events will be loosely coupled and will be more locally scoped 

## event handler in index.ts
added the event handler in the index file for easier reading

**Review by Olyno — CHANGES_REQUESTED** [review]
Hey 👋🏻 Thanks for your patience and for making another pull request! I made a first review for you. Don't hesitate to send me your feedback if you don't agree on some points, or if you have difficulties with some parts  😃

File: src/index.ts [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
-    ?.execute(interaction);
-});
+// handle discord events
+for (const event of events) {
+  if (event.once) {
+    client.once(event.name, (...args) => event.execute(...args));
+  } else {
+    client.on(event.name, (...args) => event.execute(...args));
+  }
+}
```
Comment (MagnusHJensen) [inline src/index.ts:18]: Wouldn't something like file-based discovery of events and commands be better, and less stuff to maintain?

Comment (Olyno) [reply src/index.ts:18]: Do you have an example of what you have in mind? Like the structure?

**Pull request closed without merging**